storage/plugins/book-club/src/Repo.php

```
@@ -540,6 +558,92 @@ public function createClubBook(int $clubId, int $libroId, string $state, ?int $p
+        $extId  = (int) $row['external_book_id'];
+        $titolo = (string) $row['titolo'];
+        $anno   = ($row['anno'] !== null && $row['anno'] !== '') ? (int) $row['anno'] : null;
+        $digits = $row['isbn'] !== null ? preg_replace('/[^0-9Xx]/', '', (string) $row['isbn']) : '';
+        $isbn13 = strlen((string) $digits) === 13 ? (string) $digits : null;
+        $isbn10 = strlen((string) $digits) === 10 ? (string) $digits : null;
+        $cover  = ($row['copertina_url'] !== null && $row['copertina_url'] !== '') ? substr((string) $row['copertina_url'], 0, 255) : null;
+
+        // Only `titolo` is mandatory in `libri`; everything else is optional.
+        // search_index stays NULL and is (re)built by SearchIndexBuilder on the
+        // book's next save — the row is a normal, editable catalogue entry.
+        $ok = $this->exec(
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)